Automattic/woocommerce-payments
# Add command to run QIT PHPStan tests

Closes https://github.com/Automattic/woocommerce-payments/issues/8754

#### Changes proposed in this Pull Request
This PR adds `phpstan` QIT command to be available for run.
Since both `security` and `phpstan` re-use the rest of the script, the shared code has been moved to `common.sh` while `security.sh` and `phpstan.sh` only represent specific commands. `npm` commands list was streamlined and now contains `test:qit-security` and `test:qit-phpstan`.









#### Testing instructions




* configure QIT by following instructions in `tests/qit/README.md`
* run `npm run test:qit-phpstan` and confirm the process finishes successfully and the test result link is provided at the end of execution
    * in the report, there will be 27 errors which need to be fixed and according to https://github.com/Automattic/woocommerce-payments/issues/8754's ACs, this will be done in a separate ticket
* run `npm run test:qit-security` and confirm the execution is successful

-------------------


**Post merge**
